go.mod

```diff
@@ -15,3 +16,49 @@ require (
 	gorm.io/driver/postgres v1.3.1
 	gorm.io/gorm v1.23.2
 )
+
+require (
```

Comment:
Where does these indirect deps come from? Are there all deps from go.sum? Is this generated? Do these indirect deps need to be updated manually?

Reply:
yeah this were generated by `go mod tidy` command

Reply:
found the reason - https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17?noredirect=1&lq=1

previously, the go.mod was generated on Fedora, where is still golang 1.16, 1.18 will be in Fedora 36 later this month